src/App.js

```diff
@@ -1,5 +1,5 @@
 import './App.css';
-import React from 'react';
+import React, { useState, useEffect } from 'react';
```

Comment:
remove this use Effect  and use state

Reply:
![image](https://user-images.githubusercontent.com/66599363/124502461-2d296600-dde1-11eb-86c3-5f56e022f0d3.png)

On the login page we clicking on the dark mode button it is not changing the dark color or please change the size of the button to normal.

Reply:
![image](https://user-images.githubusercontent.com/66599363/124503005-41219780-dde2-11eb-8482-dcd404edf1eb.png)

Add the dark button in here.  or please remove all the errors and warnings then make the PR @codingninja11

Reply:
If are done, then feel free to Re-request review

Reply:
> remove this use Effect  and use state 

why it should be removed?please explain!

Reply:
> ![image](https://user-images.githubusercontent.com/66599363/124503005-41219780-dde2-11eb-8482-dcd404edf1eb.png)
> 
> Add the dark button in here.  or please remove all the errors and warnings then make the PR @codingninja11

There are no errors when I'm running it on my device and for adding the functionality here please guide me how can I implement the same thing here.

Reply:
> > remove this use Effect  and use state
> 
> why it should be removed?please explain!
> @soumik2012

![image](https://user-images.githubusercontent.com/66599363/124611459-16414d00-de8f-11eb-8301-06821f8d477b.png)
see this error